shell/platform/darwin/ios/framework/Source/FlutterViewControllerTest.mm

```diff
@@ -1479,6 +1480,36 @@ - (void)testMouseSupport API_AVAILABLE(ios(13.4)) {
       dispatchPointerDataPacket:std::make_unique<flutter::PointerDataPacket>(0)];
 }
 
+- (void)testPencilSupport API_AVAILABLE(ios(13.4)) {
+  if (@available(iOS 13.4, *)) {
+    // noop
+  } else {
+    return;
+  }
```

Comment:
I don't think you need this with the `API_AVAILABLE` in the method signature.

Reply:
This section is in every other test in this file too, they also all have the API_AVAILABLE in the signature. should I remove it for the others too?

Reply:
Yes, that would be great, assuming they all pass.  Thank you!